apps/example-orchestrator-ui/components/Metadata/Products/Products.tsx

```diff
@@ -122,24 +142,24 @@ export const Products: FC<ProductsProps> = ({
     };
 
     return (
-        <>
-            <EuiSpacer size="m" />
-            <Table
-                data={data ? mapApiResponseToProductTableData(data) : []}
-                columns={tableColumns}
-                hiddenColumns={hiddenColumns}
-                dataSorting={dataSorting}
-                onDataSort={getDataSortHandler<Product>(
-                    dataDisplayParams,
-                    setDataDisplayParam,
-                )}
-                pagination={pagination}
-                isLoading={isFetching}
-                onCriteriaChange={getPageSizeHandler<Product>(
-                    setDataDisplayParam,
-                )}
-            />
-        </>
+        <TableWithFilter<Product>
```

Comment:
Providing the type Product here is not needed, I would say.

Reply:
Can't hurt either right?